Example reorg (#29)
watts-dev/watts: examples/1App_MOOSE-MultiApp_Simple/README.md

# 1App_MOOSE-MultiApp_Simple

## Purpose

> This example provides a demonstration on how to use WATTS to model simple MOOSE MultiApp model.

Reviewer: ```suggestion
This example provides a demonstration on how to use WATTS to perform a simple simulation leveraging MOOSE's MultiApps system.
```

## Code(s)
 

> - MOOSE MultiApp

Reviewer: ```suggestion
- MOOSE MultiApps
```

## Keywords
 

> - Simple MultiApp

Reviewer: ```suggestion
- Simple MultiApps
```
